go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/controller-runtime
 
-go 1.22.0
+go 1.22
```

Comment:
@fspaniol Can you try to add additional `toolchain 1.22.0` statements in both go files? I think that might solve the issue for both cases